src/Nethermind/Nethermind.Specs.Test/OverridableReleaseSpec.cs

```diff
@@ -55,7 +55,7 @@ public OverridableReleaseSpec(IReleaseSpec spec)
 
         public bool IsEip155Enabled => _spec.IsEip155Enabled;
 
-        public bool IsEip158Enabled => _spec.IsEip1559Enabled;
+        public bool IsEip158Enabled => _spec.IsEip158Enabled;
```

Comment:
Is existing logic a bug or was it intended for something?

Reply:
bug, but it was only used in tests and didn't really affect much of them except that one :)